app/src/main/java/cz/muni/fi/pv256/movio2/uco374585/ListFragment.java

```diff
@@ -0,0 +1,175 @@
+        createFakeMoviesData();
+    }
+
+    @Override
+    public void onStart() {
+        Log.i(TAG, "ListFragment is now visible to the user");
+        super.onStart();
+    }
+
+    @Override
+    public void onResume() {
+        Log.i(TAG, "ListFragment is now able to interact with the user");
+        super.onResume();
+    }
+
+    @Override
+    public void onPause() {
+        Log.i(TAG, "ListFragment is no longer interacting with the user either " +
+                "because its activity is being paused or a fragment operation is modifying it in the activity.");
+        super.onPause();
+    }
+
+    @Override
+    public void onStop() {
+        Log.i(TAG, "ListFragment fragment is no longer visible to the user either " +
+                "because its activity is being stopped or a fragment operation is modifying it in the activity.");
+        super.onStop();
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.main_fragment, container, false);
+        return view;
+    }
+
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        Button mClickButton1 = (Button) view.findViewById(R.id.miss_button);
+        mClickButton1.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                showDetail(movies.get(0));
+            }
+        });
+        Button mClickButton2 = (Button) view.findViewById(R.id.deepwater_button);
+        mClickButton2.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                showDetail(movies.get(1));
+            }
+        });
+        Button mClickButton3 = (Button) view.findViewById(R.id.rings_button);
+        mClickButton3.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                showDetail(movies.get(2));
+            }
+        });
+    }
+
+    @Override
+    public void onDestroyView() {
+        Log.i(TAG, "Here it is possible to clean up resources associated with ListFragment" +
+                "because View is being destroyed");
+        super.onDestroyView();
+    }
+
+    @Override
+    public void onDestroy() {
+        Log.i(TAG, "ListFragment is being destroyed, here clean up its resources");
+        super.onDestroy();
+    }
+
+    @Override
+    public void onDetach() {
+        Log.i(TAG, "ListFragment is being detached from activity");
+        super.onDetach();
+    }
+
+    private long convertDateToLong(String dateString) {
+        SimpleDateFormat Formatter = new SimpleDateFormat("yyyy/MM/dd");
+        Date releaseDate = new Date();
+        try {
+            releaseDate = Formatter.parse(dateString);
+        } catch (ParseException e) {
+            e.printStackTrace();
```

Comment:
lepsie je pouzivat error log, nieco ako Log.e, print stack trace asi uplne nechces v appke vsade.
to len tak btw